numpy/random/_generator.pyx

```diff
@@ -3572,13 +3572,30 @@ cdef class Generator:
         >>> y.shape
         (3, 3, 2)
 
-        Here we generate a scatter plot of 800 samples from the bivariate
-        normal distribution with mean (0, 0) and covariance matrix
-        [[6, -3], [-3, 3.5]].
+        Here we generate 800 samples from the bivariate normal distribution
+        with mean (0, 0) and covariance matrix [[6, -3], [-3, 3.5]].  The
+        expected variances of the first and second components of the sample
+        are 6 and 3.5, respectively, and the expected correlation
```

Comment:
Probably not a big deal, but the expected correlation is somewhat closer to 0 than the population correlation, due to finite sample bias in the estimator.

Reply:
It looks like the magnitude of the first term in the [correction](https://en.wikipedia.org/wiki/Pearson_correlation_coefficient#Adjusted_correlation_coefficient) would be ≈ 2.3e-4, so probably not worth mentioning in a simple example like this.